styleguide/source/assets/js/nav.js

```diff
@@ -8,48 +8,43 @@
  * - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
  */
  (function ($, Drupal) {
-   Drupal.behaviors.ribbonnav = {
+   Drupal.behaviors.nav = {
      attach: function (context, settings) {
-       (function ($) {
 
-         // Search
-
-         // When a user clicks on the ribbon trigger (main)
-         $('.ribbon_dropdown_trigger').click(function() {
+         // Menu
+         // When a user clicks on the menu trigger (main)
+         $('#menu-trigger').click(function() {
```

Comment:
We should store elements that we will call again in a variable to help with performance so that the dom does not need to be searched for it again.

Reply:
Can you please review my variables to make sure I did everything appropriately.